src/skeleplex/app/_app.py

```diff
@@ -358,3 +361,43 @@ def _on_node_selection_enabled_changed(self, enabled: bool):
             self._viewer.main_canvas.remove_skeleton_node_callback(
                 callback=self.data._on_node_selection_click,
             )
+    def add_points(self):
```

Comment:
Eventually, I would like this to be a more fully featured function that is similar to the napari add_points() method (e.g.,, one can populate the data at the same time as creating it). However, I think this is fine as is for now. We can refine in the future.